src/richie/apps/core/context_processors.py

```diff
@@ -19,6 +19,24 @@ def site_metas(request):
     """
     site_current = Site.objects.get_current()
     protocol = "https" if request.is_secure() else "http"
+    backends = getattr(settings, "LMS_BACKENDS", [])
+    lms_backends = [
```

Comment:
redundant?

Reply:
It is just to expose only used field on frontend context.